docs/python/editing.md

```diff
@@ -77,8 +88,7 @@ While editing, you can right-click different identifiers to take advantage of se
 
 ## Quick Fixes
 
-The add imports Quick Fix allows you to quickly complete import statements. First, begin by typing a package name within the editor. You will notice a Code Action is available to automatically complete the line of source code (as long as you have the module installed within the environment). Hover over the text (marked with a squiggle) and then select the Code Action light bulb when it appears. You can then select from a list of potential imports. **Note:** The functionality in the below examples is provided by the Pylance language server.
-
+The add imports Quick Fix when using Pylance allows you to quickly complete import statements. First, begin by typing a package name within the editor. You will notice a Code Action is available to automatically complete the line of source code (as long as you have the module installed within the environment). Hover over the text (marked with a squiggle) and then select the Code Action light bulb when it appears. You can then select from a list of potential imports.
```

Comment:
Will "add imports" show up as "Add Imports" in the Quick Fix UI?
Also if "Add Imports" is a string in the UI, we usually make UI strings bold.

Reply:
Hmm good point, they don't. It shows as "add <package name>":
 
![image](https://user-images.githubusercontent.com/45497113/195459639-a4f040c1-1fb9-4792-bbc9-a76c189fd016.png)

Does it make sense to reference it as `"add imports" Quick Fix`?

Reply:
I see, the UI string varies and it isn't really a command or action name so I think leaving it as "The add imports Quick Fix" is good.